account_cutoff_base/models/account_cutoff.py

```diff
@@ -109,6 +109,17 @@ def _default_cutoff_account_id(self):
         copy=False,
         check_company=True,
     )
+    auto_reverse = fields.Boolean(
+        help="Automatically reverse created move on following day. Use this "
+        "if you accrue a value end of period that you want to reverse "
+        "begin of next period",
+    )
+    move_reversal_id = fields.Many2one(
+        "account.move",
+        string="Cut-off Journal Entry Reversal",
+        readonly=True,
+        copy=False,
+    )
```

Comment:
Sorry to be late to the party, but could we make this a computed field based on `env["account.move"].search[("reversed_entry_id", "=", self.move_id)]`.

Reply:
Or `move_id.reversal_move_id and move_id.reversal_move_id[0]`